.env.template

```diff
@@ -4,3 +4,8 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
+TRELLO_TOKEN=trello-token
+TRELLO_API_KEY=trello-api-key
+BOARD_ID=trello-board-id
+COMPLETED_LIST=trello-completed-list
+NOT_STARTED_LIST=trello-not-started-list
```

Comment:
Watch out that you've changed the names of these last three variables in your code to be prefixed by `TRELLO_` so this is slightly misleading!

Reply:
Apologies, updated